src/time-slot-bar/time-slot-bar.styles.tsx

```diff
@@ -0,0 +1,129 @@
+import { ChevronLeftIcon, ChevronRightIcon } from "@lifesg/react-icons";
+import { Color } from "src/color";
+import { IconButton } from "src/icon-button";
+import { Text } from "src/text";
+import styled, { css } from "styled-components";
+import { SlotStyle } from "./types";
+
+export const CELL_WIDTH = 44; // NOTE in px
+
+export const Container = styled.div`
+    position: relative;
+    display: flex;
+    align-items: center;
+`;
+
+export const ArrowButton = styled(IconButton)<{ direction: "right" | "left" }>`
+    z-index: 2;
+    position: absolute;
+    right: ${({ direction }) => (direction === "right" ? "0rem" : "")};
+    left: ${({ direction }) => (direction === "left" ? "0rem" : "")};
+    bottom: 1rem;
+    background-color: white;
+    box-shadow: 0px 2px 8px rgba(104, 104, 104, 0.5);
+    border-radius: 50%;
+    padding: 0.5rem;
+    width: 2rem;
+    height: 2rem;
+
+    > div {
+        display: flex;
+        align-items: center;
+        justify-content: center;
+    }
+
+    &:focus {
+        -webkit-tap-highlight-color: rgba(0, 0, 0, 0);
+    }
+`;
+
+export const ArrowIconRight = styled(ChevronRightIcon)`
+    font-size: 1rem;
+    color: ${Color.Primary};
+`;
+
+export const ArrowIconLeft = styled(ChevronLeftIcon)`
+    font-size: 1rem;
+    color: ${Color.Primary};
+`;
+
+export const TimeSlotBarContainer = styled.div`
+    overflow: hidden;
+    flex-grow: 1;
+    position: relative;
+    height: 5rem;
+`;
+
+export const TimeMarkerWrapper = styled.div`
+    position: relative;
+    white-space: nowrap;
+    height: 1.25rem;
+`;
+
+export const TimeSlotWrapper = styled.div`
+    display: flex;
+    white-space: nowrap;
+`;
+
+export const TimeMarker = styled.div<{ isHour: boolean }>`
+    display: inline-block;
+    width: ${() => `${CELL_WIDTH}px`};
```

Comment:
you don't need to have a function for this. 

```suggestion
    width: ${CELL_WIDTH}px;
```